gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -872,6 +870,78 @@ public void deleteObjects(List<StorageResourceId> fullObjectNames) throws IOExce
     }
   }
 
+  private void createGrpcStubIfEnabled(GoogleCloudStorageOptions options, Credentials credentials) {
+    if (!this.storageOptions.isGrpcEnabled()) {
```

Comment:
We usually use `this.` prefix only in constructors or setters when there is a conflict between class fields and method parameters